LICENSE-binary

```diff
@@ -456,7 +456,6 @@ org.antlr:ST4
 org.antlr:stringtemplate
 org.antlr:antlr4-runtime
 antlr:antlr
-com.github.fommil.netlib:core
```

Comment:
Interesting, this wasn't in the dev/deps files to begin with. Not totally sure why, but, yes then it only needs to come out here.

Reply:
You bring up a good point: `com.github.fommil.netlib` is still a transient dependency of Spark through `org.scalanlp:breeze`. I already submitted and merged a PR there to replace the dependency (https://github.com/scalanlp/breeze/pull/811), but it hasn't been released yet.

For the sake of not regressing performance, let me re-add the dependency on `com.github.fommil.netlib:all` in `pom.xml` when enabling`-Pnetlib-lgpl` and leave a comment to explain when we can remove it.

Reply:
Oh right, breeze uses it. OK yes let's keep this aspect then.